### ISSUE TITLE
TypeError unless the Stylesheets module name is just so

### ISSUE THREAD

**Issue by jessitron** [rtfeldman/elm-css#109]
Turns out, elm css is only gonna read a module named `Stylesheets`. Leave off that s on the module name, and this is the error:

```
readme-example jessitron$ elm css src/Stylesheet.elm
Success! Compiled 3 modules.
Successfully generated /var/folders/70/gz382v_s05jbkkk2vy0p9wlw0000gn/T/tmp-86657XhOZc2kK7KVH/emitter.js
Cannot read property 'make' of undefined
Error: TypeError: Cannot read property 'make' of undefined
```

**Comment by eeue56** [comment]
Interesting! And scary. If you use the `elm-css.js`from the repo in master, do you get this same error? It's possible that the version on npm is outdated

**Comment by eeue56** [comment]
Ah, right. You need to use the `--module Stylesheet` option for this not to blow up -> https://github.com/rtfeldman/elm-css/blob/master/elm-css.js#L13

I can take a stab at changing this, there are some general improvements to the CLI tool needed anyway

**Comment by jessitron** [comment]
Failing is fine, if it gave an actionable error.

**Comment by avh4** [comment]
This appears to be fixed:  the error message in 0.6.0 is:

```
Error: I couldn't find the entry module Stylesheets.

Maybe you meant: Stylesheet
You can pass me a different module to use with --module=<moduleName>
```

**Comment by rtfeldman** [comment]
Closing in favor of #327
